src/events/navEvents.js

```diff
@@ -5,16 +5,16 @@ interview.AddEventsNavigation = function () {
 
     const loadEvaluations = function (e) {
         e.preventDefault();
-        interview.navigate('evaluations');
+        interview.modules.moduleEvaluations.init()
```

Comment:
I'd argue that the `module` prefix for the `modules` properties is redundant, since you're already pretty sure you're talking about modules, being under a namespace called `interview.modules`... Oh, yeah, and did I mention that this naming might be redundant? :stuck_out_tongue: